7zip/plan.ps1

```diff
@@ -10,6 +10,8 @@ $pkg_shasum="9bb4dc4fab2a2a45c15723c259dc2f7313c89a5ac55ab7c3f76bba26edc8bcaa"
 $pkg_filename="7z$($pkg_version.Replace('.',''))-x64.exe"
 $pkg_bin_dirs=@("bin")
 
+Write-Host 'Write-Host is a no-no.'
```

Comment:
I'm assuming this was not intentional?

Reply:
Yes this is a test commit that needs rebased out before merging.